scripts/run.sh

```diff
@@ -34,5 +34,6 @@ export ANSIBLE_HOST_KEY_CHECKING=False
 ANSIBLE_FORCE_COLOR=true ansible-playbook \
    -e "metal3_dir=$SCRIPTDIR" \
    -e "v1aX_integration_test_action=${ACTION}" \
+   -e "ansible_python_interpreter=auto" \
```

Comment:
Is it necessary to have it here when we already have it in the inventory?

Reply:
No. Good point!